.travis.yml

```diff
@@ -2,6 +2,6 @@ sudo: false
 
 language: ruby
 rvm:
-    - 2.0.0
+    - 2.2.2
```

Comment:
C'est plus maintenu 2.0.0 :+1: 
https://www.ruby-lang.org/en/news/2016/02/24/support-plan-of-ruby-2-0-0-and-2-1/